src/main.c

```diff
@@ -104,6 +103,11 @@ static int		 use_alternate_screen;
 static size_t		 query_size;
 static struct termios	 original_attributes;
 static struct choices	*choices;
+static struct {
+	size_t	size;
```

Comment:
The struct members should probably be renamed in order to match the current naming convention of pick.